app/components/calls_to_action/mailinglist_block_component.rb

```diff
@@ -0,0 +1,22 @@
+module CallsToAction
+  class MailinglistBlockComponent < ViewComponent::Base
+    attr_reader :title, :image, :title_color, :text, :link_text, :link_target
+
+    def initialize(
+      title: "Get personalised updates",
```

Comment:
```suggestion
      title: "Get personalised guidance",
```
Looking at this with fresh eyes. I don't think updates makes much sense?